src/tests/ftest/control/dmg_network_scan.yaml

```diff
@@ -6,7 +6,7 @@ timeout: 180
 server_config:
   name: daos_server
   port: 10001
-  control_log_mask: INFO
+  control_log_mask: TRACE
```

Comment:
Is this the only test to change?

Reply:
I'm thinking that the vast majority of tests don't need to run at TRACE level because that stuff doesn't really change from run to run. I'd like to see how things go with the proposed logging updates and we can continue to adjust going forward. I'm absolutely willing to accept that this PR may have gone too far in the opposite direction, but I think we won't really know until we try it.

Reply:
My instinct is to run with TRACE as a default, since the main use of test logs is debugging failures (which may be hard to reproduce).

Reply:
I get that, but the way I'm looking at it is that running at DEBUG by default in CI forces us to be mindful of what gets logged. If we're running at TRACE all the time for all tests even though it's just noise 99% of the time, it doesn't force us to make those decisions. Honestly, I'd kind of like to run at INFO for more tests, because it will again force us to think more carefully about what should be logged in production systems.

That having been said, I'm totally in favor of finding some useful subset of tests where we should run with TRACE because those tests are exercising the network probing stuff or are otherwise specifically testing agent/hardware logic and having the extra logging is likely to yield critical details.